R/analytics.R

```diff
@@ -36,15 +36,14 @@ analyticsUI <- function(id){
 #' @noRd
 analyticsServer <- function(id){
   moduleServer(id, function(input, output, session){
-    if (Sys.getenv("s3") != ""){
-      print(Sys.getenv("s3"))
-      s3 <- Sys.getenv("s3")
-      s3_data <- s3$get_object(Bucket = "aiworld", Key = "dataset/dataset.RDS")
-      s3_train <-s3$get_object(Bucket = "aiworld", Key = "dataset/train.RDS")
-      s3_test <- s3$get_object(Bucket = "aiworld", Key = "dataset/test.RDS")
-      data <- "data.rds"
-      train <- "train.rds"
-      test <- "test.rds"
+    if (exists("aws_s3", inherits = T)){
+      aws_s3_inst <- get("aws_s3", inherits = T)
+      s3_data <- aws_s3$get_object(Bucket = "aiworld", Key = "dataset/dataset.RDS")
+      s3_train <-aws_s3$get_object(Bucket = "aiworld", Key = "dataset/train.RDS")
+      s3_test <- aws_s3$get_object(Bucket = "aiworld", Key = "dataset/test.RDS")
+      data <- "../tmp/data.rds"
```

Comment:
Because `/tmp` in root dir of docker liara PaaS cloud is the only writable dur, we must add temp files to this dir, so it avoids `read-only` error.